tests/common.go

```diff
@@ -474,6 +474,13 @@ type PlatformCredentialStruct struct {
 	credUID  string
 }
 
+type (
+	// TestcaseOwner represents the owner of a Testcase
+	TestcaseOwner string
```

Comment:
Make it `TestcaseAuthor`

Reply:
The term 'Owner' is used in the sheets, and I prefer because the author and owner might eventually be different.

Reply:
Done.